priv/graphql/schemas/admin/cets.gql

```diff
@@ -1,11 +1,14 @@
 "Allow admin to get information about CETS status"
 type CETSAdminQuery @protected{
-    "Get from the local node. Only for global admin"
-    systemInfo: [CETSInfo]
+    "Get a list of tables from the local node. Only for global admin"
```

Comment:
I would skip the "Only for global admin" notes, because:
- they appear in `mongooseimctl cets`, where they have no meaning,
- other commands don't have such information,
- the `@protected` directive already gives this information.